pkg/validation/internal/operatorhub.go

```diff
@@ -267,6 +275,13 @@ func checkAnnotations(checks CSVChecks) CSVChecks {
 			checks.errs = append(checks.errs, fmt.Errorf("csv.Metadata.Annotations.Capabilities %s is not a valid capabilities level", capability))
```

Comment:
Unrelated to this PR, but I noticed in the test you added that the word "Magic" was not quoted, which made it harder to deciper, for me at least:
```suggestion
			checks.errs = append(checks.errs, fmt.Errorf("csv.Metadata.Annotations.Capabilities %q is not a valid capabilities level", capability))
```

Reply:
Another similar nit for the categories message in lines 298 and 306.

Also in line 306, WDYT about `s/default/standard/`?